Makefile

```diff
@@ -40,6 +40,9 @@
 #
 ##############################################################################
 
+# FIXME ideally we would define this only once between here and libdeflate.h
+VERSION=1.8
```

Comment:
How about getting the version number from libdeflate.h using `grep`?

Reply:
cheerfully done. i ended up using sed for reasons that ought be obvious upon inspection.